db/migrate/20150922165007_create_ex_links.rb

```diff
@@ -0,0 +1,15 @@
+class CreateExLinks < ActiveRecord::Migration
+  def change
+    create_table :ex_links do |t|
+      t.string :label
+      t.text :url
+      t.string :tags
```

Comment:
Does it mean [serialization](http://api.rubyonrails.org/classes/ActiveRecord/AttributeMethods/Serialization/ClassMethods.html)?